src/CreditCard/CreditCardForm.js

```diff
@@ -111,10 +160,12 @@ const CreditCardForm = ({ card, onchange, handleChangeCode, handleChangeDate, ha
       <div>
         <fieldset className={Style.info_user}>
           <label>MM/YY</label>
-          <input 
-            type="text"
+          <MaskedInput
+            type='tel'
```

Comment:
Type maybe should be date?

Reply:
No, si se pone type date no funciona el mask